router.go

```diff
@@ -196,7 +196,11 @@ type APIResponse struct {
 // Write - Reponse interface implementation
 func (res APIResponse) Write(w http.ResponseWriter, r *http.Request) {
 	if res.Status == "ERROR" {
-		logger.Errorf("[API][PATH: %s]:: Error handling request. ERROR: %s. User agent: %s", r.RequestURI, res.Error, r.Header.Get("User-Agent"))
+		logger.Errorf("API - PATH: %s:: Error handling request. ERROR: %s. User agent: %s", r.RequestURI, res.Error, r.Header.Get("User-Agent"))
```

Comment:
why are we changing this format? [API][PATH:? It might impact the existing dashbaord.

@abhyuday cc